wikipron/cli.py

```diff
@@ -104,6 +104,12 @@ def _get_cli_args(args: List[str]) -> argparse.Namespace:
         dest="tone",
         help="removes tones from the transcriptions",
     )
+    parser.add_argument(
+        "--keep-parens",
+        action="store_true",
+        dest="parens",
+        help="include parentheses in the transcriptions",
```

Comment:
I see the other flags are described in the repo's top-level `README.md`, under the "Usage" section. Probably a note on `--keep-parens` would be needed there as well.

Reply:
+1